src/main/java/com/eerussianguy/firmalife/init/FoodDataFL.java

```diff
@@ -6,4 +6,26 @@
 {
     public static final FoodData CHOCOLATE = new FoodData(4, 0.0F, 0.2F, 0.0F, 0.0F, 0.0F, 0.0F, 0.2F, 1.1F);
     public static final FoodData COCOA_BEANS = new FoodData(4, 0.0F, 0.1F, 0.1F, 0.0F, 0.0F, 0.1F, 0.0F, 0.5F);
+    //TODO: balance dried fruits (currently apply a 1/3 modifier to decay rate)
+    public static final FoodData DRIED_BANANA = new FoodData(4, 0.0F, 0.2F, 0.0F, 1.0F, 0.0F, 0.0F, 0.0F, 0.66F);
```

Comment:
For your FoodData instances, it's fine to leave it like this, but my pattern so far (which is motivated by how I construct ItemFoodFL) is to use generic food data, as to avoid repeated instantiation. So like, you could have DRIED_FRUIT_WEAK or something to that effect instead of repeating it. In general I find that more readable / easier to document than a massive list.